src/components/frame/LandingPage/LandingPage.tsx

```diff
@@ -1,125 +1,169 @@
+      iconName: 'colony-icon',
+      onClick: () => navigate('/colony/meta'),
+      disabled: true,
+    },
+  ];
 
   return (
-    <div className={styles.main}>
-      <div>
-        <div className={styles.title}>
+    <div className="w-full">
+      <div className="mb-8">
+        <div className="flex items-center mb-4">
           <Heading
-            text={MSG.callToAction}
-            appearance={{ size: 'medium', margin: 'none', theme: 'dark' }}
+            text={MSG.headerTitle}
+            className="font-semibold text-gray-900 text-3xl"
+          />
+          <span className="font-medium text-blue-400 text-sm px-3 py-1 bg-blue-100 rounded-3xl ml-3">
+            <FormattedMessage {...MSG.privateBetaLabel} />
+          </span>
+        </div>
+        <p className="text-normal text-gray-600">
+          <FormattedMessage {...MSG.headerDescription} />
+        </p>
+      </div>
+      <div className="w-full flex justify-center gap-4  max-w-[1286px]">
+        <div className="w-1/2 flex flex-col gap-4">
+          {landingPageItems.map((item, index) => (
+            <LandingPageItem
+              {...item}
+              itemIndex={index}
+              onHover={setHoveredItem}
+            />
+          ))}
+        </div>
+        <div className="w-1/2 bg-gray-100 rounded-lg max-h-[395px]">
+          <img
+            src={
+              landingPageItems[hoveredItem].imgSrc ?? landingPageItems[0].imgSrc
+            }
+            className="w-full h-full"
+            alt=""
           />
         </div>
-        <ul>
-          {wallet && !userLoading && !user && (
-            <LandingItem to={CREATE_USER_ROUTE} message={MSG.createUsername} />
-          )}
-          {canInteractWithNetwork && (
-            <LandingItem to={CREATE_COLONY_ROUTE} message={MSG.createColony} />
-          )}
-          {loading && (
-            <li className={styles.itemLoading}>
-              <SpinnerLoader appearance={{ size: 'medium' }} />
-            </li>
-          )}
-          {metacolony && (
-            <li className={styles.item}>
-              <NavLink
-                to={`/colony/${metacolony.name}`}
-                className={styles.itemLink}
-              >
-                <ColonyAvatar
-                  className={styles.itemIcon}
-                  colonyAddress={metacolony.colonyAddress}
-                  colony={metacolony}
-                  size="xl"
-                />
-                <span className={styles.itemTitle}>
-                  <FormattedMessage
-                    {...MSG.exploreColony}
-                    values={{
-                      colonyName:
-                        metacolony.metadata?.displayName || metacolony.name,
-                    }}
-                  />
-                </span>
-              </NavLink>
-            </li>
-          )}
-        </ul>
       </div>
+      {/* @TODO: Add real logic here to display the invitation block */}
```

Comment:
Same as the above comment if the user has a private invite code attached to it you can show this invitation block